langchain/package.json

```diff
@@ -838,8 +838,8 @@
     "@faker-js/faker": "^7.6.0",
     "@getmetal/metal-sdk": "^4.0.0",
     "@getzep/zep-js": "^0.9.0",
-    "@gomomento/sdk": "^1.44.1",
-    "@gomomento/sdk-core": "^1.44.1",
+    "@gomomento/sdk": "^1.47.1",
+    "@gomomento/sdk-core": "^1.47.1",
```

Comment:
Bumps the dev dependencies for `@gomomento/sdk-core` and `@gomomento/sdk`. We need `sdk-core` because it contains the interface types we code the integration against. We need `@gomomento/sdk` for the integration tests.